EdvClient.js

```diff
@@ -63,46 +66,48 @@ export class EdvClient {
    * they contain that attribute. Compound indexes can be specified by
    * providing an array for `attribute`.
    *
-   * @param {Object} options - The options to use.
+   * @param {object} options - The options to use.
    * @param {string|Array} options.attribute - The attribute name or an array of
    *   attribute names to create a unique compound index.
-   * @param {boolean} [options.unique=false] Should be `true` if the index is
+   * @param {boolean} [options.unique=false] - Should be `true` if the index is
    *   considered unique, `false` if not.
    */
   ensureIndex({attribute, unique = false}) {
-    return this.indexHelper.ensureIndex({attribute, unique});
+    this.indexHelper.ensureIndex({attribute, unique});
   }
 
   /**
+   * @typedef Readable
```

Comment:
this probably should be `ReadableStream`. We also might not need the typedef if `ReadableStream` is imported in this file.